reconciler/reconciler_test.go

```diff
@@ -36,7 +39,7 @@ var _ = Describe("Reconciler", func() {
 			Flags:     []string{"flag-1", "flag-2"},
 		},
 		HealthCheck: &types.VirtualService_HealthCheck{
-			Endpoint:      "http:102/health",
+			Endpoint:      &wrappers.StringValue{Value: "http:102/health"},
```

Comment:
Is this supposed to be invalid, or just an oversight?

Reply:
Health checker is mocked out so it doesn't matter what it's set to. But I can fix it.